app/lib/frontend/handlers/routes.dart

```diff
@@ -201,6 +201,11 @@ class PubSiteService {
   Future<Response> oauthCallback(Request request) async =>
       oauthCallbackHandler(request);
 
+  /// List of the current user's publishers.
+  @Route.get('/account/publishers')
```

Comment:
Maybe we should re-think this pattern.

Reply:
++, luckily it will be an easy thing to change